utilities/network.py

```diff
@@ -1113,3 +1114,39 @@ def verify_dhcpd_activated(vm):
     except TimeoutExpiredError:
         LOGGER.error(f"{dhcpd} status is not '{active}' but rather '{sample}'")
         raise
+
+
+def verify_nmstate_bridge_creation(bridge: LinuxBridgeNodeNetworkConfigurationPolicy) -> None:
+    # Although the bridge interface was already created, the NodeNetworkState resource takes some time to be updated.
+    bridge_name = bridge.bridge_name
+    sampler = TimeoutSampler(
+        wait_timeout=TIMEOUT_1MIN,
+        sleep=5,
+        func=lambda: NodeNetworkState(
+            name=utilities.infra.get_node_selector_name(node_selector=bridge.node_selector)
```

Comment:
This is flaky. If this function is called after creating an NNCP without any node-selector, it will break.